domenetjenester/mottak/src/main/java/no/nav/foreldrepenger/mottak/kompletthettjeneste/KompletthetssjekkerInntektsmelding.java

```diff
@@ -27,11 +31,32 @@ public KompletthetssjekkerInntektsmelding(InntektsmeldingRegisterTjeneste inntek
 
 
     public List<ManglendeVedlegg> utledManglendeInntektsmeldinger(BehandlingReferanse ref, Skjæringstidspunkt stp) {
-        return inntektsmeldingRegisterTjeneste.utledManglendeInntektsmeldingerFraAAreg(ref, stp)
+        //Bruker tjeneste som også vurderer om søkers arbeidsforhold har relevant permisjon.
+        var manglendeInntektsmeldingerUtenRelevantPermisjon = finnManglendeInntektsmeldingerUtenRelevantPermisjon(ref, stp);
+
+        var manglendeInntektsmeldinger = inntektsmeldingRegisterTjeneste.utledManglendeInntektsmeldingerFraAAreg(ref, stp)
             .keySet()
             .stream()
             .map(it -> new ManglendeVedlegg(DokumentTypeId.INNTEKTSMELDING, it.getIdentifikator()))
             .toList();
+
+        if (!manglendeInntektsmeldingerUtenRelevantPermisjon.isEmpty() && !manglendeInntektsmeldingerUtenRelevantPermisjon.stream().sorted().toList().equals(manglendeInntektsmeldinger.stream().sorted().toList())) {
```

Comment:
Denne testen kan lett snuble på rekkefølge i lista. Her sorterer du men et greit mønster for å sammenligne to lister / sett er å sjekke l1.size() == l2.size() && l1.containsAll(l2)